### DISCUSSION TITLE
Integrating spring-rdf4j with allegrograph

### DISCUSSION THREAD

**Discussion by ranjith1235** [eclipse-rdf4j/rdf4j#4845] [Q&A]
I am trying to integrate with allegrograph but getting error for INSERT. SELECT statement is working fine. Below is the body of the request which is being sent to allegrograph. 
`<transaction>
<sparql includeInferred='true'>
<updateString>INSERT { ?aggregationSet_id a &lt;http://geli.net/AggregationSet&gt; ;
&lt;http://geli.net/name&gt; ?aggregationSet_name . }
WHERE {}</updateString>
<bindings>
<binding_uri name='aggregationSet_id'>urn:uuid:bdbf8a15-ae68-4ca5-a5fc-0352c375f08a</binding_uri>
<binding_literal name='aggregationSet_name' dataType='http://www.w3.org/2001/XMLSchema#string'>testset</binding_literal>
</bindings>
</sparql>
</transaction>
`
I am getting a 400 bad request with `Stray 'INSERT DATA { ?aggregationSet_id a ' in document.`
The same works with inmemory.enabled set to true. Any idea on whats happening here.

**Comment by hmottestad** [comment]
@dancyatfranz could you get someone from Franz to help out?
